src/core/lib/compression/compression_internal.h

```diff
@@ -73,6 +77,7 @@ class CompressionAlgorithmSet {
   // Add algorithm to this set.
   void Set(grpc_compression_algorithm algorithm);
 
+  size_t GzipCompressionForLevel() const;
```

Comment:
I don't see this method defined anywhere.  And it probably does not make sense to have this be part of `CompressionAlgorithmSet`, since this is the options to use with a given algorithm, not part of the set of algorithms.